docs/_news/news06.md

```diff
@@ -18,7 +18,7 @@ Thursday 21 November 2019
 - In VS Code, to skip the **Select a parent directory** step when you create a project, use the `codewind.alwaysCreateProjectsInWorkspace` setting.
 - Project information is now kept in the `codewind_cw-workspace` Docker volume. Information for existing projects is moved to the Docker volume when Codewind 0.6.0 is started for the first time. If this volume is deleted, your projects are deleted from Codewind. However, they still exist on your disk, and you can add them back into Codewind with the `Add Existing Project` command.
 - Whenever files in the project are changed, the changed files are copied into the `codewind_cw-workspace` Docker volume. Previously, Codewind did a Docker mount of the entire `codewind-workspace`.
-**Eclipse**
+**Eclipse**<br>
```

Comment:
Use new lines to create that space instead of inserting HTML. For example:

For your PR, I noticed Eclipse was still stuck to the last bullet point. This is what I did:
Docker mount of the entire codewind-workspace.

**Eclipse**

The information in the **VS Code** section also applies to Eclipse but with these exceptions: